e2e/ansible/playbooks/hyperconverged/test-resize-jiva-volume/resize-volume.yml

```diff
@@ -0,0 +1,281 @@
+          register: result
+          delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+          until: "'Running' in result.stdout"
+          delay: 30
+          retries: 5
+
+        - name: Get $HOME of K8s master for kubernetes user
+          shell: source ~/.profile; echo $HOME
+          args:
+            executable: /bin/bash
+          register: result_kube_home
+          delegate_to: "{{groups['kubernetes-kubemasters'].0}}" 
+ 
+        - name: Copy the percona definition yaml to k8s master
+          copy:
+            src: "{{ percona_def }}"
+            dest: "{{ result_kube_home.stdout }}"
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+        - name: Getting the node name
+          shell: kubectl get nodes -o wide  | awk {'print $1'} | awk 'FNR == 4{print}'
+          args:
+            executable: /bin/bash
+          register: node_name
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+  
+        - name: Creating label for the node
+          shell: kubectl label nodes {{node_name.stdout}} app=percona
+          args:
+            executable: /bin/bash
+          register: label
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+          until: '"labeled" in label.stdout'
+          delay: 10
+          retries: 3
+
+        - name: Deploying application
+          shell: source ~/.profile; kubectl apply -f "{{ percona_def }}"
+          args:
+            executable: /bin/bash
+          register: percona_status
+          delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+          until: "'deployment \"percona\" created' in percona_status.stdout"
+          delay: 30
+          retries: 5
+        
+        - name: Checking if the application is deployed successfully
+          shell: kubectl get pods --all-namespaces | grep percona
+          args:
+            executable: /bin/bash
+          register: app_status
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+          until: "'Running' in app_status.stdout"
+          delay: 30
+          retries: 15
+    
+        - name: Getting PV name
+          shell: kubectl get pv | grep openebs-percona | awk {'print $1'}
+          args:
+            executable: /bin/bash
+          register: pv
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+         
+        - name: Getting the controller service IP address
+          shell: kubectl get svc | grep "{{ pv.stdout }}" | grep ctrl | awk {'print $3'}
+          args:
+            executable: /bin/bash
+          register: target_IP
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0}}"
+         
+        - name: Getting the device name
+          shell: fdisk -l | grep "5 G" | awk {'print $2'} | sed 's/.$//'
+          args: 
+            executable: /bin/bash
+          become: true
+          register: device_name
+          delegate_to: "{{groups['kubernetes-kubeminions'].1}}"
```

Comment:
Can we get the device node upon login from iscsi module itself ? That will be more foolproof